helloworld.java

```diff
@@ -0,0 +1,3 @@
+public class a{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class name 'a' doesn't match '[A-Z][a-zA-Z0-9]*'](https://app.codacy.com/app/gagsdeep/TestSonar/pullRequest?prid=4663122)